src/scripts/Tasks.js

```diff
@@ -1,27 +1,29 @@
-
-
 export class Tasks {
-  constructor() {
+  constructor () {
     this.taskList = [
       { index: 1, desc: 'task1', done: false },
       { index: 2, desc: 'task2', done: false },
-      { index: 3, desc: 'task3', done: false }
+      { index: 3, desc: 'task3', done: false },
+      { index: 4, desc: 'task4', done: false },
```

Comment:
- [ ] I will encourage you to always prioritize project requirements to your personal thoughts, the project requirement states that you use description, not desc, also completed not done. Kindly fix those up, outside these, you did a wonderful work.

Reply:
ok all done